app/javascript/articles/Article.jsx

```diff
@@ -0,0 +1,136 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import { articlePropTypes } from '../src/components/common-prop-types/article-prop-types';
+import {
+  CommentsCount,
+  ContentTitle,
+  OrganizationHeadline,
+  PublishDate,
+  ReadingTime,
+  SaveButton,
+  SearchSnippet,
+  TagList,
+  ReactionsCount,
+} from './components';
+import { PodcastArticle } from './PodcastArticle';
```

Comment:
All these components above would probably never be used on their own. I broke them up like this when building the `<Article />` based off of what was in `buildArticleHTML.js.erb`. They're all imported because code climate was complaining about file size.